templates/github/.github/workflows/release.yml.j2

```
@@ -76,7 +134,7 @@ jobs:
       - name: Publish docs to pulpproject.org
         run: bash .ci/scripts/publish_docs.sh tag $BRANCH
         env:
-            - BRANCH=${GITHUB_REF##*/}
+          BRANCH: ${GITHUB_REF##*/}
```

I'd prefer
```suggestion
          BRANCH: ${GITHUB_REF#refs/*/}
```
or even
```suggestion
          BRANCH: ${GITHUB_REF#refs/{heads,tags}/}
```
Maybe the variable deserves another name now...